com.blunix.git.currency_converter.yml

```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.blunix.git.currency_converter
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: currency_converter
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  # currency_converter
+  - name: currency_converter
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        url: https://git.blunix.com/miscellaneous/currency-converter/uploads/2583116834138e76d8c22e684303cb04/currency_converter-Linux-Portable.tar.gz
```

Comment:
how does that work on aarch64?

Reply:
so what happend in the aarch64 flatpak that is built?

Reply:
I have not conducted builds or tests on aarch64 devices as such I cannot provide insight about the apps performance on that architecture. Currently it only supports x86_64 architectures

Reply:
that still doesn't answer the question. Let me rephrase again:

There is an aarch64 flatpak being built too. This will not work. You must disable the aarch64 build. This is not done.

Reply:
I understood now, apologies for the oversight. 
I added {"only-arches": ["x86_64"] } in flathub.json to disable aarch64 build on flathub.